### DISCUSSION TITLE
Organization Account Flagged 2 Weeks, no answer from support

### DISCUSSION THREAD

**Discussion by rodrigoventuring** [community/community#128030] [Enterprise] [answered]
### Body

It's been two weeks since my organization was flagged, breaking a lot of integrations, repo and project managment, etc. I have already raised a ticket to support requesting it's reinstatement, but got no answers.
Is there any way to rise visibility on this issue to somebody on GitHub's side?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @rodrigoventuring, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
